src/pages/patch/ActionButtons.tsx

```diff
@@ -24,8 +32,21 @@ export const ActionButtons: React.FC<ActionButtonProps> = ({
   const wrapperRef = useRef(null);
   const [isVisible, setIsVisible] = useState(false);
   const { id: patchId } = useParams<{ id: string }>();
-
   const [isActionLoading, setIsActionLoading] = useState(false);
+  const { successBanner, errorBanner } = useBannerDispatchContext();
+  const refetchQueries = ["Patch"];
```

Comment:
i don't think this needs to be its own variable - it can be dynamically stated on like 48: `refetchQueries: ["Patch"]`. There is no perf gain from putting it as a variable outside of the mutation.

Reply:
fixed